assets/report.json

```diff
@@ -4546,10 +4525,14 @@
     "flash.display.FocusDirection": {
```

Comment:
This class is AIR-only and should be removed from the list.

Reply:
Removed from the avm2 spec